### ISSUE TITLE
Incorrect test rule output

### ISSUE THREAD

**Issue by DarthVi** [antlr/intellij-plugin-v4#334]
I expect the message `line 1:0 mismatched input '12' expecting {<EOF>, BINDIGIT}` with input `12 + 20 - `applied to the following grammar. This is the correct behaviour I can obtain by using antlr from command line.

However by using the "test rule" dialog from the plugin inside Intellij IDEA I get no errors and an incorrect parse tree.

![screenshot_20180620_155046](https://user-images.githubusercontent.com/8702259/41662591-be3cad48-74a1-11e8-8f63-0fe693663f2e.png)

```
grammar GEN_18_2018;

start   :  (BINDIGIT PIU NUMBER MENO)*  ;
PIU :    '+'    ;
MENO    :   '-' ;
BINDIGIT    :   ('0'|'1')+  ;
NUMBER  :     ('0'..'9')+   ;
WS      : (' '|'\t'|'\n'|'\r')-> skip   ;
```

Plugin version: 1.8.4.
IntelliJ IDEA version: Ultimate 2018.1, build IU-181.5087.20

**Comment by bjansen** [comment]
I can't reproduce the problem, on my machine the IntelliJ plugin outputs the same thing as the CLI: nothing.

```
$ java org.antlr.v4.Tool GEN_18_2018.g4
$ javac GEN_18_2018*.java
$ cat foo.txt
12
$ java org.antlr.v4.gui.TestRig GEN_18_2018 start foo.txt
$
```

However, if I replace `(BINDIGIT PIU NUMBER MENO)*` with `(BINDIGIT PIU NUMBER MENO)+` I get this error:

```
line 1:0 mismatched input '12' expecting BINDIGIT
```

Tested on both ANTLR 4.7 (shipped in plugin version 1.8.4) and 4.7.2 (shipped in the current plugin).

**Comment by bjansen** [comment]
OTOH if I change the `start` rule to `(BINDIGIT PIU NUMBER MENO)*  EOF` as @ericwen229 suggested, I get this:

```
line 1:0 extraneous input '12' expecting {<EOF>, BINDIGIT}
``` 

So the plugin works like the CLI.
